### PR TITLE
Add 'keybase pgp list' command

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

**Comment by jacobhaven** [comment]
lgtm.

File: go/client/cmd_pgp_list.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			continue
+		}
+		dui.Printf("Keybase Key ID:  %s\n", key.KID)
+		dui.Printf("PGP Fingerprint: %s\n", libkb.PGPFingerprintFromHexNoError(key.PGPFingerprint).ToQuads())
+		if len(key.PGPIdentities) > 0 {
+			dui.Printf("PGP Identities:\n")
+			for _, id := range key.PGPIdentities {
+				var comment string
+				if len(id.Comment) > 0 {
+					comment = fmt.Sprintf(" (%s)", id.Comment)
```
Comment (oconnor663) [inline go/client/cmd_pgp_list.go]: We could just use `Printf` here too?

**Comment by oconnor663** [comment]
:ship:

File: go/client/cmd_pgp_list.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	if err != nil {
+		return err
+	}
+
+	dui := s.G().UI.GetDumbOutputUI()
+	for _, key := range publicKeys {
+		if len(key.PGPFingerprint) == 0 {
+			continue
+		}
+		dui.Printf("Keybase Key ID:  %s\n", key.KID)
```
Comment (oconnor663) [inline go/client/cmd_pgp_list.go]: One more bikeshedding thought: we've unfortunately used "key_id" to refer to a shortened fingerprint in a few places, like [in the sigchain](https://keybase.io/oconnor663/chain). That be a reason to prefer "KID" here, even though it's less clear by itself? Not sure.

File: go/client/cmd_pgp_list.go (hunk 1 above)
Comment (patrickxb) [inline go/client/cmd_pgp_list.go]: No, it's building up a string.

File: go/client/cmd_pgp_list.go (hunk 2 above)
Comment (patrickxb) [inline go/client/cmd_pgp_list.go]: This is what dump-keyfamily does, so I'm going with this for now...

**Pull request merged**